README.md

```diff
@@ -6,10 +6,17 @@
 Makes it easy to integrate your [Spree][1] app with [MailChimp][2].
 
 **List synchronization**
-> Automatically syncs Spree's user list with MailChimp. The user can subscribe/unsubscribe via the registration and account pages.
+> Automatically syncs Spree's user list with MailChimp. The user can
+> subscribe/unsubscribe via the registration and account pages.
 
 **Order synchronoization**
-> Fully supports MailChimp's [eCommerce360][3] API. Allows you to create targeted campaigns in MailChimp based on a user's purchase history. We'll even update MailChimp if the order changes after the sale (i.e. order modification, cancelation, return).
+> Fully supports MailChimp's [eCommerce360][3] API. Allows you to
+> create targeted campaigns in MailChimp based on a user's purchase history.
+> We'll even update MailChimp if the order changes after the
+> sale (i.e. order modification, cancelation, return). User's who check out
+> with their email in the Spree Storefront, will accrue order data under this
+> email in MailChimp. This data will be available under the 'E-Commerce' tab
```

Comment:
Love this change. Much clearer what's going on. Lot's of folks seem to trip up with what this extension actually does.